packages/mongoose/src/generateModel.ts

```diff
@@ -101,7 +101,7 @@ export const generateSchema = <T>(
 						populatedVirtuals = [],
 						virtuals = [],
 						methods = {}
-					} = recursivelyGenerateSchema(classType, null);
+					} = recursivelyGenerateSchema(classType);
```

Comment:
passing null was an expedient to avoid propagating the default options to sub-schemas, as commented [here](https://github.com/HPInc/davinci/blob/f387963d6af05ce56252cb14f564203ece41f300/packages/mongoose/src/generateModel.ts#L97)

As an alternative solution we can explicitly pass the options (like timestamps: false)

Reply:
Yes I saw a test failing about that. I refactored the code and now that should be handled.
I also added tests